server/task_executor_pitr_cutover.go

```diff
@@ -73,18 +73,19 @@ func (exec *PITRCutoverTaskExecutor) RunOnce(ctx context.Context, server *Server
 	return terminated, result, nil
 }
 
+// The PITR cutover algorithm is:
```

Comment:
nit: double-check this section. Use Go style, comment starting with function name.

// pitrCutover will xxx.

Reply:
SG.